examples/QmlBarcodeGenerator/qml/GeneratorPage.qml

```diff
@@ -213,40 +221,67 @@ ApplicationWindow {
         CTextField {
           id: widthField
 
+          implicitWidth: parent.width
+          implicitHeight: parent.height / 8
+
           placeholderText: "Current width: " + barcodeGenerator.width
 
-          onEditingFinished: barcodeGenerator.width = parseInt(text)
+          onEditingFinished: function () {
+            if (parseInt(text)) {
+              barcodeGenerator.width = parseInt(text)
```

Comment:
Instead of doing the parsing twice you can do simply parse it once and then check if parsing was successful using `isNan` function: https://developer.mozilla.org/pl/docs/Web/JavaScript/Reference/Global_Objects/Number/isNaN